engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -223,6 +224,13 @@ class WorkflowActor(val workflowId: WorkflowId,
     case Event(WorkflowExecutionFailedResponse(executionStore, outputStore, failures),
     data @ WorkflowActorData(_, Some(workflowDescriptor), _, _)) =>
       finalizeWorkflow(data, workflowDescriptor, executionStore, outputStore, Option(failures.toList))
+    case Event(msg @ EngineStatsActor.JobCountQuery, data) =>
```

Comment:
This would also address the concern in my "aborting versus aborted" comment below